### ISSUE TITLE
Add "integrity" attribute to "Copy script tag"

### ISSUE THREAD

**Issue by ahmetb** [cdnjs/new-website#181]
Ay plans to `integrity="<hash>"` attribute to the "Copy script tag" functionality?

For example:

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"
  integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb"
  crossorigin="anonymous"></script>
```

**Comment by PeterDaveHello** [comment]
@ahmetb wanna send a pull request to add it?

**Comment by dakshshah96** [comment]
@PeterDaveHello, can I help with this? :)

**Comment by ahmetb** [comment]
Looks like it's fixed now. We can close this.

**Comment by PeterDaveHello** [comment]
Partially solved only, see #152 :)
